modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -309,6 +311,8 @@ Status description:: error: invalid transaction termination - inner transactions
 [[transaction-rollback]]
 == Transaction rollback
 
+Transaction rollback errors occur when the transaction is rolled back due to an error, such as when the transaction rollback fails, or when the constituent rollback fails.
```

Comment:
This sentence is a little bit hard to understand. Maybe it is just my lacking English, but to me it sounds a bit like the transaction was rolled back because of an error and that error was that the rollback failed. That cannot be the intended meaning though, because that would be a circle dependency.